.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
```

Comment:
I have objection to add pre-commit here. A workflow to add them has already been discussed and documented (see jira associated ticket)
This kind of sneaky commit only add more mess into each repo (ex: adding a lot of python pre-commit hook, when you remove all the python code in the repo ...?)

Reply:
@fblackburn1 Are you refering to [this ticket](https://wazo-dev.atlassian.net/browse/WAZO-2984)? 
What is the documented workflow?

Reply:
No I was talking about [WAZO-2982](https://wazo-dev.atlassian.net/browse/WAZO-2982) (I add a ref to your mentioned ticket)
But let's talk about it together it will be easier to explain :)